recipe/meta.yaml

```diff
@@ -23,6 +23,9 @@ requirements:
   host:
     - python >=3.6
     - pip
+    - setuptools >=45
+    - setuptools-scm >=7
+    - tomli >=1.0.0
```

Comment:
Added these to line up with [upstream build requirements]( https://github.com/conda-forge/conda-smithy/blob/a5a45bbdc0fae354345abb85404050f61a16a306/pyproject.toml#L2-L6 ) included in this release